.github/workflows/publish.yml

```diff
@@ -0,0 +1,61 @@
+name: publish
+
+on:
+  push:
+    branches: [master]
+    tags: ['*']
+
+jobs:
+  publish-scm-1:
+    if: github.ref == 'refs/heads/master'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: tarantool/rocks.tarantool.org/github-action@master
+        with:
+          auth: ${{ secrets.ROCKS_AUTH }}
+          files: expirationd-scm-1.rockspec
+
+  publish-tag:
+    if: startsWith(github.ref, 'refs/tags/')
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - run: printf '%s=%s\n' TAG "${GITHUB_REF##*/}" >> "${GITHUB_ENV}"
+
+
+      # `tarantoolctl rocks pack <rockspec>` creates a source
+      # tarball. It speeds up
+      # `tarantoolctl rocks install <module_name> <version>` and
+      # frees it from dependency on git.
+      #
+      # Important: Don't confuse this command with
+      # `tarantoolctl rocks pack <module_name> [<version>]`, which
+      # creates a **binary** rock or .all.rock (see [1]). Don't
+      # upload a binary rock of a Lua/C module to
+      # rocks.tarantool.org. Lua/C modules are platform dependent.
+      #
+      # A 'pure Lua' module is packed into the .all.rock tarball.
+      # Feel free to upload such rock to rocks.tarantool.org.
+      # Don't be confused by the 'pure Lua' words: usage of
+      # LuaJIT's FFI and tarantool specific features are okay.
+      #
+      # [1]: https://github.com/luarocks/luarocks/wiki/Types-of-rocks
+      - uses: tarantool/setup-tarantool@v1
+        with:
+          tarantool-version: '2.5'
+
+      # Create a rockspec for the release.
+      - run: tarantoolctl rocks new_version --tag ${{ env.TAG }}
+
+      # Create a .all.rock tarball.
+      - run: tarantoolctl rocks install expirationd-${{ env.TAG }}-1.rockspec
+      - run: tarantoolctl rocks pack expirationd ${{ env.TAG }}
+
+      # Upload .rockspec and .src.rock.
```

Comment:
We upload .rockspec and .all.rock here: the comment is misleading.

BTW, is there any sense to upload .src.rock as well, when we upload .all.rock? (I don't know.)

Reply:
> BTW, is there any sense to upload .src.rock as well, when we upload .all.rock? (I don't know.)

Asked @rosik and @olegrok: they don't know cases, when it may be needed.

Reply:
Fixed the comment.